### PR TITLE
PR for (header-with-styles)

### PR DESCRIPTION
**Pull request by HamzaAnwar1998**
created a .gitignore file
set up linters for html css
created an index.html file and committed it
created a styles.css file then committed it
created a test.md file with some text
rules added to ignore test.md file

pushed the code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (medaminedev66) [inline .gitignore]: - [ ] Following the project requirments, You should create a `test.md` file and ignore it by git `(gitignore)`.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @HamzaAnwar1998😃, wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- The repo looks professional✔️
- Git/GitHub flow best practices have been followed✔️
- You have a descriptive PR✔️

## Required Changes ♻️
- [ ] Please ensure that your project doesn't have Linters errors. A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Check this link for more information [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (HamzaAnwar1998) [reply .gitignore]: Do you mean to add all files to commit using git add . and then see if test.md is tracked or untracked?

**Review by Deepakdanger — APPROVED** [review]
Hi @HamzaAnwar1998, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**